stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/FilterObject.kt

```diff
@@ -0,0 +1,33 @@
+package io.getstream.chat.android.client.api.models
+
+public sealed class FilterObject
```

Comment:
What will the usage of these new classes look like? Are clients expected to create these directly?

Reply:
It Will be used by `Filters` class, but it will be cover in another PR

Reply:
Should/could all these classes be `internal` then, at least for now?

Reply:
Ok, I can open on the next PR